server/src/transactions/transactions.service.ts

```diff
@@ -30,6 +32,16 @@ export class TransactionsService {
     return found;
   }
 
+  // TODO make omit user its own type
+  async getCurrentUsersTransactions(user: Omit<User, 'password'>): Promise<Transaction[]> {
+    const found = await this.transactionsRepository.find({
+      where: [{ recipient: user.organizations }, { asset: user.assets }],
```

Comment:
I think you need to first get the assets belonging to this user, then use the asset ID to find its transactions

Reply:
reworked this a bit, lmk what you think. tested and confirmed that user not included in transaction cannot see that transaction (and included in test plan)